soql-standalone-parser/src/main/scala/com/socrata/soql/tokens/Tokens.scala

```diff
@@ -18,6 +18,7 @@ sealed abstract class ValueToken[T] extends Token {
 }
 
 // Keywords
+// TODO: why are all of these case classes w/ 0 params instead of case objects?....
```

Comment:
Be the change you want to see.

Reply:
The answer is that they're mutable.  They all have a `position` which set set post-construction.